components/dropdown/src/auro-dropdownBib.js

```diff
@@ -10,10 +10,20 @@ import styleCss from "./styles/bibStyles-css.js";
 import colorCss from "./styles/bibColors-css.js";
 import tokensCss from "./styles/tokens-css.js";
 
+
+const DESIGN_TOKEN_BREAKPOINT_PREFIX = '--ds-grid-breakpoint-';
```

Comment:
I wonder if there is a way we can future proof this to reduce the risk we break the code if the breakpoint token names change in the future. Let's open an investigation token to consider this as a future possible improvement.

Reply:
created #81